plugins/callback/timestamp.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, kurokobo <kurokobo@protonmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+  name: timestamp
+  type: stdout
+  short_description: Adds simple timestamp for each header
+  version_added: 9.0.0  # for collections, use the collection version, not the Ansible version
+  description:
+    - This callback adds simple timestamp for each header
+  author: kurokobo (@kurokobo)
+  options:
+    timezone:
+      description: timezone to use for the timestamp in IANA time zone format (for example C(America/New_York), C(Asia/Tokyo)). Ignored on Python < 3.9
+      ini:
+        - section: callback_timestamp
+          key: timezone
+      env:
+        - name: ANSIBLE_CALLBACK_TIMESTAMP_TIMEZONE
+      default: ""
```

Comment:
```suggestion
      type: string
```
The empty string is NOT a valid time zone, so it should not be used as a default.

Reply:
TIL that if `default` is not defined here, the actual default is null